bootstrapform_jinja/templatetags/bootstrap.py

```diff
@@ -1,110 +1,148 @@
-from django import forms
+from django.forms import CheckboxInput, CheckboxSelectMultiple, \
```

Comment:
I prefer parenthesis my self too, but used that format for some reason. I'll check the 120 limit too.